src/test/java/com/myhome/services/integration/HouseMemberDocumentServiceIntegrationTest.java

```diff
@@ -0,0 +1,270 @@
+package com.myhome.services.integration;
+
+import com.myhome.controllers.integration.ControllerIntegrationTestBase;
+import com.myhome.controllers.request.CreateUserRequest;
+import com.myhome.controllers.request.LoginUserRequest;
+import com.myhome.domain.HouseMember;
+import com.myhome.domain.HouseMemberDocument;
+import com.myhome.repositories.HouseMemberDocumentRepository;
+import com.myhome.repositories.HouseMemberRepository;
+import com.myhome.repositories.UserRepository;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Wildcard imports.
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

Please setup IntelliJ correctly
https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

Reply:
Thanks